guides/hack/09-asynchronous-operations/14-concurrent.md

```diff
@@ -0,0 +1,28 @@
+Expected to launch with HHVM 4.5, it'll be possible to use `concurrent` blocks. `concurrent` blocks are a replacement for the `Tuple\from_async` pattern that makes the behavior of the code much clearer. It concurrently awaits all `await`s within a `concurrent` block and it works with [`await`-as-an-expression](await-as-an-expression.md) as well!
+
+## Syntax
+
+```
+concurrent {
+  $x = await genx();
+  await gen_void();
+  $sum = await geny() + await genz();
+}
+$y = $x + $sum;
+```
+
+## Order-of-execution
+
+Each of the statements in a concurrent block should be thought of as running concurrently (*surprise*?). This means there shouldn't be any depedencies between the statements in a `concurrent` block.
```

Comment:
- '*surprise*' probably shouldn't still be here
- it's useful to either avoid the words concurrent/concurrency, or to add a reminder each time that `await` is not multithreading
- dependencies spelling